packages/@icon-magic/config-reader/package.json

```diff
@@ -14,15 +14,12 @@
     "prebuild": "rm -rf ./dist",
     "build": "tsc -p tsconfig.json",
     "postbuild": "cp -r ./test/fixtures ./dist/test",
-    "lint": "tslint -t msbuild --project . -c tslint.json"
+    "lint": "tslint -t msbuild --project . -c tslint.json --fix"
```

Comment:
Its nice to have separate `lint` and `lintfix` tasks :)

Reply:
Added a lintfix script